velox/functions/sparksql/Arithmetic.h

```diff
@@ -262,6 +262,16 @@ struct Log1pFunction {
   }
 };
 
+
+struct Expm1Function {
+  FOLLY_ALWAYS_INLINE void call(double& result, double a) {
+    // The std::expm1 is more accurate than the expression std::exp(num) - 1.0 if num is close to zero.
+    // Spark uses Java's StrictMath we need to keep it as below.
```

Comment:
Did you mean that Spark uses java.lang.StrictMath.expm1 ?

This sentence is still "broken". Perhaps, change to something like

```
This matches Spark's implementation that uses java.lang.StrictMath.expm1.
```